ember-server/shared/src/main/scala/org/http4s/ember/server/internal/ServerHelpers.scala

```diff
@@ -507,6 +505,7 @@ private[server] object ServerHelpers extends ServerHelpersPlatform {
       }
       .takeWhile(_.headers.get[Connection].exists(_.hasKeepAlive))
       .drain
+      .mask
```

Comment:
So masking the Stream suppresses the errors and allows the Stream to successfully end without letting errors bubble up. I think this is a good idea. It makes the Stream more robust.